CMakeLists.txt

```diff
@@ -163,4 +99,58 @@ install(FILES
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Arcus
 )
 
-include(CPackConfig.cmake)
+# Create the Python bindings
+if(BUILD_PYTHON)
+    list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+
+    if(NOT DEFINED Python_VERSION)
+        set(Python_VERSION
+                3.10
+                CACHE STRING "Python Version" FORCE)
+        message(STATUS "Setting Python version to ${Python_VERSION}. Set Python_VERSION if you want to compile against an other version.")
+    endif()
+    if(APPLE)
+        set(Python_FIND_FRAMEWORK NEVER)
+    endif()
+    find_package(cpython ${Python_VERSION} QUIET COMPONENTS Interpreter Development)
+    if(NOT TARGET cpython::cpython)
+        find_package(Python ${Python_VERSION} EXACT REQUIRED COMPONENTS Interpreter Development)
+    else()
+        add_library(Python::Python ALIAS cpython::python)
+        set(Python_SITEARCH "${CMAKE_INSTALL_PREFIX}/lib/python3.10/site-packages")
```

Comment:
`Python_VERSION` also holds the patch version should be `/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages`

I will change that

Reply:
Nope won't changed yet. This was written if we use the Conan package cpython. Currently that package doesn't set the `Python_VERSION(_<MAJOR,MINOR,PATCH>)` This is however something that will be addressed when working on CURA-8787.
See this CPython recipe which will serve as the basis for CURA-8787 https://github.com/Ultimaker/conan-ultimaker-index/tree/master/recipes/cpython

Reply:
`Python_VERSION` defaults to `3.10` as written in line 110. I don't expect that it would contain the patch version as well.

If it is a problem to change due to the way Conan exports it, let's leave it as-is.